<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-1LI4ATB" failures="1" tests="2" name="Default test(failed)" time="0.037" errors="0" timestamp="01 Oct 2020 18:11:00 GMT">
  <testcase classname="demo.TestNGRetryFailedDemo" name="test2" time="0.009"/>
  <testcase classname="demo.TestNGRetryFailedDemo" name="test3" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:26)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="demo.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
